### ISSUE TITLE
Are V2 sensors now supported?

### ISSUE THREAD

**Issue by yusefasjana** [kevinvincent/ha-wyzesense#215]
Hi,

@skilau  I saw this [commit](https://github.com/kevinvincent/ha-wyzesense/commit/1fdee5c721947534328853b583daa361a23e73ca#diff-4a17e39edb60055029b678d22fb62828c3a799ad44c89e7c9f27732980b8f4fa) and  I was wondering if that means that now V2 sensors can be added to home assistant or is this the start of that feature. 

Just double checking before I buy some V2 sensors

Thanks in advance

**Comment by skilau** [comment]
Yes, both v2 (motion and door) sensors are supported now by this driver.

However, the new v2 standalone Hub is *NOT*, only the old v1 USB Dongle will work with this driver.
So don't bother buying the new HUB package, just buy the v2 sensors only.

**Comment by ryanwnzl** [comment]
Thanks skilau. I just ordered some of the new v2 contact sensors to include on my v1 USB Dongle.

Does anyone know if Wyze Sense Leak Sensor Probes are supported?

**Comment by yusefasjana** [comment]
> Yes, both v2 (motion and door) sensors are supported now by this driver.
> 
> However, the new v2 standalone Hub is _NOT_, only the old v1 USB Dongle will work with this driver.
> So don't bother buying the new HUB package, just buy the v2 sensors only.

**Comment by skilau** [comment]
> Does anyone know if Wyze Sense Leak Sensor Probes are supported?

Not yet.

I have contemplated buying them to add support, but I already have settled on using the Govee Water Leak sensors instead.
(Mostly because Wyze didn't have their sensors yet when I bought the Govees, and I didn't know when Wyze would finally release them!)

**Comment by tony-fav** [comment]
I ordered some of the Leak Sensors and Temp/Humidity Sensors as I really would like those things on a supported cloud rather than my hacky HA setup. They don't start shipping until August. I'm happy to help. @skilau do you have a process outlined for how you found the id code for the V2 contact or just trial and error?

**Comment by skilau** [comment]
> 
> 
> I ordered some of the Leak Sensors and Temp/Humidity Sensors as I really would like those things on a supported cloud rather than my hacky HA setup. They don't start shipping until August. I'm happy to help. @skilau do you have a process outlined for how you found the id code for the V2 contact or just trial and error?

For the v2 Door/Motion sensors, I just turned on full debugging in the driver, and checked what it was dumping out.
That is how I found out that it was the same protocol, and all Wyze had done was changed the id's.

I suspect the Water Leak sensor will be the same, as it really is just a binary on/off switch, like the existing door/motion sensors, so it should be trivial to add support for.

But the Temp/Humidity sensors probably will need the most help/work, as it is actually sending more data payload across, with the temp and humidity values.
So I am guessing that that sensor will require you/us to figure out how they embedded those values in the new/additional payload.
